schemas/input/csl-data.json

```
@@ -58,6 +58,9 @@
       "id": {
         "type": ["string", "number"]
       },
+      "citekey": {
```

```suggestion
      "citation-key": {
```
See comment below.

I'm just proposing; not insisting ;-)